packages/components/src/Workflow/_docs/Workflow.mdx

```diff
@@ -0,0 +1,82 @@
+import { LinkTo } from "../../../../../storybook/components/LinkTo"
+import * as Workflow from "./Workflow.stories"
+import * as WorkflowFooter from "./WorkflowFooter.stories"
+import * as WorkflowHeader from "./WorkflowHeader.stories"
+import { KaioNotification } from "../../../../../storybook/components/DocsContainer/ComponentDocsTemplate/subcomponents/KaioInstallation/KaioNotification"
+
+<Meta title="Workflow" />
+
+# Workflow
+
+<ul className="!list-none !p-0 flex gap-4 items-center mb-8">
+  <li className="!mt-0 !text-paragraph">
+    <a
+      href="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/Workflow"
+      target="_blank"
+      rel="noopener noreferrer nofollow"
+    >Source Code</a>
+  </li>
+</ul>
+
+<KaioNotification />
+
+<div>
+  <h2 id="installation">Installation</h2>
+  <Source code="yarn add @kaizen/components" language="tsx" />
+  <Source code='import { Workflow } from "@kaizen/components"' language="tsx" />
+</div>
+
+## Playground
+
+This is a page template component containing the header, footer and main landmarks that compose a Workflow page. Its purpose is to guide an customer through a multi-step form to create a Workflow.
+
+The Worflow is intended as a page wrapper and sets minimum height of `100vh` with a sticky `Header` component. The `Children` will be wrapped in a unstyled main landmark to provide flexibility for inner content layouts.
+
+This component has responsiveness baked in to support 400% zoom and smaller screen sizes. This will also un-stick the `Header` to provide more realestate for the user.
+
+<Canvas layout="fullscreen" of={Workflow.Playground} />
+
+## Header actions
+
+The `headerActions` prop  takes array of JSX elements that will render in the top right of the `Header` component.
+
+While the number of JSX element is not limited, is important to consider the realestate in the `Header` and how larger strings or components may be rendered on smaller screen sizes.
+
+<Canvas layout="fullscreen" of={WorkflowHeader.MultipleActions} />
+
+
+## Handling a Workflow exit
+
+This package also exports a `WorkflowExit` component that can be  used to handle a customer leaving a Workflow page.
+
+This triggers a modal that can provide context to a customers and confirm thier intention to exit the workflow. The `onExit` callback is triggered by a button labelled with the `confirmExitLabel` prop.
+
+<Canvas layout="fullscreen" of={WorkflowHeader.Playground} />
+
+## Tracking progress
+
+The Footer tracks the progress of the form by comparing the current step, indicated by `stepName`, against its position in the `steps` array.
+
+<Canvas layout="fullscreen" of={WorkflowFooter.FirstStep} />
+
+The Footer is agnostic to the JSX elements that are used in the `previousAction` and `nextAction`. We recommend using the `Button` component but an alternative JSX component can be used if you require NextJS Routing.
```

Comment:
> if you require NextJS Routing.

[Not entirely true](https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3128133092/Solution+Preview+support+for+custom+routing) 😉

Reply:
Fair point, I could rephrase it to:
> While we recommend using the Kaizen `Button`, a button-like component can be used in its place to satisfy project-specific requirements